UPGRADE-7.0.md

```diff
@@ -102,16 +102,67 @@ The code entities in this list were marked as deprecated in previous releases an
 
 ## JavaScript Files and Functions
 
+- bootstrap-colorpicker (new version 3.0.3 located in src/CoreBundle/Resources/public/javascript/jquery/bootstrap-colorpicker-3.0.3).
+- chosen.jquery.js
```

Comment:
Could you also remove all those libs from the THIRD-PARTY-LICENSE file?

Reply:
done